common/test/acceptance/tests/lms/test_learner_profile.py

```diff
@@ -71,7 +79,6 @@ def visit_profile_page(self, username, privacy=None):
 
         # Load the page
         profile_page.visit()
-        profile_page.wait_for_page()
```

Comment:
reviewer note: .visit already does a .wait_for_page at the end.
See https://github.com/edx/bok-choy/blob/53eee41959365357da0f6040f50a6bb5b1bf38c5/bok_choy/page_object.py#L285